htdocs/scss/components/quick-reply.scss

```
@@ -183,9 +183,12 @@
             flex-grow: 1;
             flex-shrink: 1;
         }
-        #comment-text-quote {
-            margin-left: 5px;
-        }
+    }
+
+    .qr-markup {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-end;
```

```suggestion
        align-items: flex-end;
        
        & > div {
            display: inline-block;
        }
```

probably ought to throw this in there for non-flexy browsers, so they'll still display side-by-side.